src/plugins/saved_objects/public/saved_object/saved_object_loader.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
For new files we should use the simplified header:

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/b2cfb6e9d54dc75dbc739da9a4caa1006b3d2ff5/.eslintrc.js#L34-L39

Reply:
thanks, done